### PR TITLE
File naming

### PR DESCRIPTION
**Pull request by tschmidtb51**
- resolves secvisogram/secvisogram#177
- update regex for file naming

### CODE REVIEW

File: app/lib/SecvisogramPage.js [hunk 1]
```diff
@@ -385,7 +385,7 @@ function createFileName(doc, isValid, extension) {
   if (trackingId.trim().length === 0) {
     trackingId = 'csaf_2_0'
   } else {
-    trackingId = trackingId.replace(/([^a-z0-9]+)/gi, '_')
+    trackingId = trackingId.replace(/([^a-z0-9+\-_]+)/gi, '_')
```
Comment (sthagen) [inline app/lib/SecvisogramPage.js:388]: Nit: Why would we want to replace an underscore with itself?

**Review by sthagen — APPROVED** [review]

File: app/lib/SecvisogramPage.js (hunk 1 above)
Comment (tschmidtb51) [reply app/lib/SecvisogramPage.js:388]: Actually, we don't. But as the regex starts with `^` after the `[` the `_` should not match...

Comment (sthagen) [reply app/lib/SecvisogramPage.js:388]: Silly me, yes, obviously 😅

Comment (tschmidtb51) [reply app/lib/SecvisogramPage.js:388]: No worries!

**Review by mfd2007 — APPROVED** [review]

**Pull request merged**